plugins/woocommerce/client/admin/client/settings-payments/hooks/use-payment-ordering.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { useState, useEffect } from 'react';
+
+const usePaymentOrdering = ( id: string ) => {
+	const [ ordering, setOrdering ] = useState< Record< string, number > >(
+		{}
+	);
+	const [ loading, setLoading ] = useState( true );
+	const [ error, setError ] = useState< unknown >( null );
+
+	useEffect( () => {
+		const fetchOrdering = async () => {
+			try {
+				setLoading( true );
+				const storedOrdering = localStorage.getItem(
```

Comment:
This will eventually need to be replaced with an API call and a backend, spike pending: pe4Hk0-1KO-p2